src/03_modules.py

```diff
@@ -10,25 +10,30 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-for arg in sys.argv:
-    print(arg)
+print("The CLI Argument is: " + str(sys.argv))
 
 # Print out the OS platform you're using:
 # YOUR CODE HERE
 print(sys.platform)
 
 # Print out the version of Python you're using:
 # YOUR CODE HERE
-
+print(sys.version)
 
 import os
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
 # YOUR CODE HERE
+pid = os.getpid()
+print (pid)
 
 # Print the current working directory (cwd):
 # YOUR CODE HERE
+cwd = os.getcwd()
+print (cwd)
 
 # Print out your machine's login name
 # YOUR CODE HERE
```

Comment:
im getting os has no attribute uname, i think you can just do os.getlogin()

Reply:
got it, will change.